tutorials/boundary_conditions.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.            return values.ravel("F")
</pre><p>Could you please add a comment about the "F" convention in PorePy?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>